components/molecule/imageEditor/demo/articles/ArticleCustomChildren.js

```diff
@@ -0,0 +1,43 @@
+import {useState} from 'react'
+
+import PropTypes from 'prop-types'
+
+import {Article, H2, Paragraph, Code} from '@s-ui/documentation-library'
+
+import MoleculeImageEditor from '../../src/index.js'
+import {DEMO_IMAGE} from '../settings.js'
+import CustomChildren from './CustomChildren.js'
+
+const ArticleCustomChildren = ({className}) => {
+  const [croppedImage, setCroppedImage] = useState(DEMO_IMAGE)
+
+  return (
+    <Article className={className}>
+      <H2>Custom Children</H2>
```

Comment:
```suggestion
      <H2>Customise the layout using a custom Children</H2>
```

I think this title is easier to understand